.github/workflows/upgrade-python-requirements.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     uses: openedx/.github/.github/workflows/upgrade-python-requirements.yml@master
     with:
        branch: ${{ github.event.inputs.branch }}
-       team_reviewers: "arbi-bom"
-       email_address: arbi-bom@edx.org
+       team_reviewers: "orbi-bom"
+       email_address: orbi-bom@edx.org
```

Comment:
I believe you'll want the email address to be for JSM instead, and possibly to include upgrade-python-requirements as scope. Would you want to test a change like this in JSM before rolling out further?